### ISSUE TITLE
Best hyperparams for finetune yolov8 models

### ISSUE THREAD

**Issue by flywheel1412** [ultralytics/ultralytics#3977]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Thanks for your wonderful work. This is awesome.

I'm trying to finetune the pretrained `rtdetr-l.pt` on custom dataset in default hyperparams, the mAP%.5 is less than yolov5m by 10+%.
could you give some advices about hyperparams for finetune pretrained yolov8 models?

**Comment by glenn-jocher** [comment]
@flywheel1412 hello and thanks for appreciating the Ultralytics YOLOv8 framework! The success of this project is due not only to the hard work of the Ultralytics team, but also to the shared wisdom and contributions of the YOLO community.

Fine-tuning pretrained models on custom datasets is highly dependent on the dataset and the use case. Hence, there is no one-size-fits-all solution when it comes to hyperparameters. However, some general rules of thumb might help:

1. **Try to decrease the learning rate**: If your finetuning doesn't give you the results you want, try decreasing the learning rate since a slower learning rate might help the model to generalize better.

2. **Increase the number of epochs**: If your model is not overfitting, increasing the number of training epochs might help.

3. **Modify weight decay**: A lower value can sometimes improve model performance.

One of the key strengths of YOLOv8 is its automated hyperparameters tuning that can help you find the optimal combination without manual trial and error. Be sure to take advantage of that.

I hope these suggestions help you in your work, and I encourage you to share your results with the community to contribute to the continuous improvement of YOLOv8!

Best regards,
Glenn Jocher and the Ultralytics team

**Comment by cpingan1201** [comment]
> 你好，感谢您对 Ultralytics YOLOv8 框架的赞赏！这个项目的成功不仅归功于 Ultralytics 团队的辛勤工作，还归功于 YOLO 社区的共享智慧和贡献。
> 
> 在自定义数据集上微调预训练模型在很大程度上取决于数据集和使用案例。因此，在超参数方面没有万能的解决方案。但是，一些一般的经验法则可能会有所帮助：
> 
> 1. **尝试降低学习率**：如果您的微调没有为您提供所需的结果，请尝试降低学习率，因为较慢的学习率可能有助于模型更好地泛化。
> 2. **增加 epoch 的数量**：如果您的模型没有过拟合，则增加训练 epoch 的数量可能会有所帮助。
> 3. **修改权重衰减**：较低的值有时可以提高模型性能。
> 
> YOLOv8 的主要优势之一是其自动超参数优化，可以帮助您找到最佳组合，而无需手动试错。一定要利用这一点。
> 
> 我希望这些建议对你的工作有所帮助，并鼓励你与社区分享你的成果，为 YOLOv8 的持续改进做出贡献！
> 
> 此致 Glenn Jocher 和 Ultralytics 团队

I took your advice and reduced the learning rate, I also froze the backbone module by freeze parameter, or more layers, but training for 2-5 epochs, the recognition of the coco dataset decreases a lot (model.val for coco's validation set), is it catastrophic oblivion, and how can I solve this problem

**Comment by glenn-jocher** [comment]
@cpingan1201 reducing the learning rate and freezing layers can sometimes lead to underfitting, especially if the model's performance decreases. Consider unfreezing some layers and gradually increasing the learning rate. Additionally, ensure your dataset is well-prepared and balanced. For more detailed guidance, you might find our [training tips](https://docs.ultralytics.com/guides/model-training-tips/) helpful.
